definitions/checks/remote_execution/verify_settings_file_already_exists.rb

```diff
@@ -0,0 +1,42 @@
+module Checks::RemoteExecution
+  class VerifySettingsFileAlreadyExists < ForemanMaintain::Check
+    metadata do
+      description 'Check to verify remote_execution_ssh settings already exist'
+      tags :pre_upgrade
+
+      confine do
+        feature(:downstream) &&
```

Comment:
@iNecas,

Could you please suggest what conditions should be needed in confine block?
Is this check applicable for upstream scenario as well?

Reply:
I believe this is downstream only, so I'm quite happy with the confine block I see right now.